flink-end-to-end-tests/test-scripts/common_yarn_docker.sh

```diff
@@ -129,21 +129,31 @@ END
     docker exec master bash -c "cat /home/hadoop-user/$FLINK_DIRNAME/conf/flink-conf.yaml"
 }
 
-function copy_and_show_logs {
-    mkdir -p $TEST_DATA_DIR/logs
+function debug_copy_and_show_logs {
+    echo "Debugging failed YARN Docker test:"
+    echo "Currently running containers"
+    docker ps
+
+    echo "Currently running JVMs"
+    jps -v
+
     echo "Hadoop logs:"
-    docker cp master:/var/log/hadoop/* $TEST_DATA_DIR/logs/
-    for f in $TEST_DATA_DIR/logs/*; do
+    mkdir -p $TEST_DATA_DIR/logs
+    docker cp master:/var/log/hadoop/ $TEST_DATA_DIR/logs/
+    ls -lisah $TEST_DATA_DIR/logs/hadoop
+    for f in $TEST_DATA_DIR/logs/hadoop/*; do
         echo "$f:"
         cat $f
     done
+    
     echo "Docker logs:"
     docker logs master
 
     echo "Flink logs:"
     docker exec master bash -c "kinit -kt /home/hadoop-user/hadoop-user.keytab hadoop-user"
     docker exec master bash -c "yarn application -list -appStates ALL"
     application_id=`docker exec master bash -c "yarn application -list -appStates ALL" | grep "Flink" | grep "cluster" | awk '{print \$1}'`
+    
```

Comment:
I added these blank lines to have more visual structure in this method (which as grown a bit)